### PR TITLE
chore: sync with the poc repo

### PR DESCRIPTION
**Pull request by brionmario**
## Purpose

The initial POC for the Oxygen UI Design System was done in https://github.com/brionmario/oxygen-ui-poc and it's now ready to be moved in to WSO2 space.

## Goals

- Sync with the POC repo.

### CODE REVIEW

File: packages/primitives/scripts/build.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
```
Comment (thivi) [inline packages/primitives/scripts/build.js:1]: Is this needed?

File: packages/react/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline packages/react/.storybook/main.js:2]: Shall we update the license year?

File: packages/react/src/components/Button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ * under the License.
+ */
+
+import React, {FC, ReactElement} from 'react';
+import clsx from 'clsx';
+import MuiButton, {LoadingButtonProps as MuiButtonProps} from '@mui/lab/LoadingButton';
+import {WithWrapperProps} from '../../models';
+import {composeComponentDisplayName} from '../../utils';
+
+
```
Comment (thivi) [inline packages/react/src/components/Button/Button.tsx:25]: Shall we remove this extra line?

File: packages/react/.storybook/theme.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline packages/react/.storybook/theme.js:2]: Shall we update the year?

File: packages/react/.storybook/story-config.ts [hunk 5]
```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline packages/react/.storybook/story-config.ts:2]: Shall we update this year?

File: packages/react/.storybook/preview.tsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (thivi) [inline packages/react/.storybook/preview.tsx:2]: Shall we update this year?

File: packages/react/.storybook/manager.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline packages/react/.storybook/manager.js:2]: Shall we update this year?

**Review by thivi — DISMISSED** [review]

File: .vscode/extensions.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
```
[the remark this answers is not in the document]
Comment (brionmario) [reply .vscode/extensions.json:1]: We can add recommended VS code extensions here.

[the remark this answers is not in the document]
Comment (brionmario) [reply .vscode/extensions.json:1]: It's actually suggested to commit some of the .vscode files for better consistent IDE support.

Ref: https://stackoverflow.com/a/57749909/7833500

File: packages/primitives/scripts/build.js (hunk 1 above)
Comment (brionmario) [reply packages/primitives/scripts/build.js:1]: This script is supposed to be an executable. Do you have any concerns here?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+
+Follow this guide to set up the source code, development tools & other software.
+
+* [Mandatory Software & Tools](#mandatory-software-and-tools)
+* [Setting up Development Tools](#setting-up-development-tools)
+* [Setting up the Source Code](#setting-up-the-source-code)
+
+### Mandatory Software & Tools
+
+To build and write code, make sure you have the following set of tools on your local environment.
```
Comment (savindi7) [inline CONTRIBUTING.md:19]:
```suggestion
To build and write code, make sure you have the following set of tools in your local environment.
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,53 @@
...
+  <a href="./LICENSE"><img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License"></a>
+
+  <br>
+  <br>
+  🚧 This is still a conceptual project. Please do not use this in production 🚧
+</div>
+
+<br>
+
+Oxygen UI is the underline design system that powers WSO2's core products like Asgardeo, Choreo, WSO2 Identity Server, etc.
```
Comment (savindi7) [inline README.md:22]:
```suggestion
Oxygen UI is the underlying design system that powers WSO2's core products like Asgardeo, Choreo, WSO2 Identity Server, etc.
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,53 @@
...
+
+This repository contains the source code of the key components that works together for building resilient UIs.
+
+## Packages
+
+- [`@oxygen-ui/primitives`](./packages/primitives) - Low level building blocks of Oxygen UI.
```
Comment (savindi7) [inline README.md:28]: WDYT about including examples like this?
```suggestion
- [`@oxygen-ui/primitives`](./packages/primitives) - Low level building blocks of Oxygen UI (e.g. icons, fonts) 
```

File: README.md [hunk 12]
```diff
@@ -1,2 +1,53 @@
...
+- [`@oxygen-ui/react`](./packages/react) - The React implementation of Oxygen UI.
+- [`@oxygen-ui/logger`](./packages/logger) - Logger for the Oxygen UI packages
+
+## Examples
+
+* [💅 Multi Brand Identity Demo](https://oxygen-multi-brand-example.vercel.app/)
+
+    Sample app to showcase Oxygen Design System' multi-branding capabilities.
```
Comment (savindi7) [inline README.md:36]:
```suggestion
    Sample app to showcase Oxygen Design System's multi-branding capabilities.
```

File: docs/styles/fonts.ts [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {Space_Grotesk} from '@next/font/google';
+
+export const spaceGrotesk = Space_Grotesk({subsets: ['latin']});
```
Comment (savindi7) [inline docs/styles/fonts.ts:21]: Shall we use Gilmer for this also? wdyt

**Review by savindi7 — APPROVED** [review]

File: examples/multi-brand-identity/src/branding/themes/ballerina.ts [hunk 14]
```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (brionmario) [reply examples/multi-brand-identity/src/branding/themes/ballerina.ts:2]: The file was actually created in 2022. I'm moving the repo with the commit history. So it should be 2022.

File: packages/react/.storybook/main.js (hunk 2 above)
Comment (brionmario) [reply packages/react/.storybook/main.js]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: packages/react/.storybook/theme.js (hunk 4 above)
Comment (brionmario) [reply packages/react/.storybook/theme.js]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: packages/react/.storybook/preview.tsx (hunk 6 above)
Comment (brionmario) [reply packages/react/.storybook/preview.tsx]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: packages/react/.storybook/manager.js (hunk 7 above)
Comment (brionmario) [reply packages/react/.storybook/manager.js]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: docs/styles/fonts.ts (hunk 13 above)
Comment (brionmario) [reply docs/styles/fonts.ts:21]: +1. I'll send a follow up PR.

File: README.md (hunk 12 above)
Comment (brionmario) [reply README.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/1/commits/d13b674f11e96733251d16a3b8bc439870821b6e

File: README.md (hunk 11 above)
Comment (brionmario) [reply README.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/1/commits/d13b674f11e96733251d16a3b8bc439870821b6e

File: README.md (hunk 10 above)
Comment (brionmario) [reply README.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/1/commits/d13b674f11e96733251d16a3b8bc439870821b6e

File: CONTRIBUTING.md (hunk 9 above)
Comment (brionmario) [reply CONTRIBUTING.md]: Fixed with https://github.com/wso2/oxygen-ui/pull/1/commits/d13b674f11e96733251d16a3b8bc439870821b6e

File: packages/react/src/components/Button/Button.tsx (hunk 3 above)
Comment (brionmario) [reply packages/react/src/components/Button/Button.tsx]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: packages/react/.storybook/story-config.ts (hunk 5 above)
Comment (brionmario) [reply packages/react/.storybook/story-config.ts]: Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6

File: packages/primitives/scripts/build.js (hunk 1 above)
Comment (thivi) [reply packages/primitives/scripts/build.js:1]: If it is an executable, then we can go ahead with this.

File: .vscode/settings.json [hunk 15]
```diff
@@ -0,0 +1,38 @@
+{
```
[the remark this answers is not in the document]
Comment (brionmario) [reply .vscode/settings.json:1]: Removed the `workbench.colorCustomizations` as discussed offline. https://github.com/wso2/oxygen-ui/pull/1/commits/9af18bc61954a1daedd6a5b35635bd5891e2b3c0

**Review by thivi — APPROVED** [review]

**Pull request merged**